hbase-server/src/main/java/org/apache/hadoop/hbase/master/MasterRpcServices.java

```diff
@@ -1499,6 +1492,17 @@ public RunCatalogScanResponse runCatalogScan(RpcController c,
     }
   }
 
+  @Override
+  public RunHbckChoreResponse runHbckChore(RpcController c, RunHbckChoreRequest req)
+      throws ServiceException {
+    rpcPreCheck("runHbckChore");
+    HbckChore hbckChore = master.getHbckChore();
+    if (!hbckChore.isRunning()) {
```

Comment:
If already running, should result of this call be 'false' for did not run?//

Reply:
Ok. Return false if the HBCK chore already running.